hack/build-image.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/bash
 
-set -euxo pipefail
+if [ "${DEBUG:-}" = "true" ]; then
```

Comment:
Not v important but FYI "${DEBUG:-}" is equivalent to "$DEBUG".

Reply:
but "$DEBUG" does not allow you default a value is that not correct?

Reply:
right, `${DEBUG:-}` would default to `` if `DEBUG` isn't set